p2p/libp2p/directSender.go

```diff
@@ -27,6 +27,8 @@ var _ p2p.DirectSender = (*directSender)(nil)
 
 const timeSeenMessages = time.Second * 120
 const maxMutexes = 10000
+const signPrefix = "en-directsend:"
```

Comment:
maybe use the same prefix as in libp2p so that we can treat broadcast and direct send messages similarly.

Reply:
fixed, reused pubsub.SignPrefix